lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1902,6 +1902,15 @@ bool LLVMToSPIRVBase::transDecoration(Value *V, SPIRVValue *BV) {
   }
   transMemAliasingINTELDecorations(V, BV);
 
+  if (auto *CI = dyn_cast_or_null<CallInst>(V)) {
```

Comment:
Why do we expect `V` to be `nullptr` here?

Reply:
Right, I have no reason to expect it to be `nullptr`. I'll fix it.